mautic/mautic
# Add segment build time measurement for each segment

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
This is follow up for https://github.com/mautic/mautic/pull/9298
Just add time of run rebuild for each segment. It help fast monitor long-run segments. 

![image](https://user-images.githubusercontent.com/462477/123830104-4a52c600-d903-11eb-98f3-5b943f0c6287.png)




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create few segments
3. Run `php bin/console m:s:r --timing=1`
4. You should see time of each segment in console as image above




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10208"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
